packages/data-mate/src/validations/field-validator.ts

```diff
@@ -386,12 +386,12 @@ export function isValidDate(input: any): boolean {
 
 // NOTE: this function will throw compared to all other validations
 export function guard(input: any) {
-    if (input === undefined) throw new Error('input is empty');
+    if (ts.isNil(input)) throw new Error('input is empty');
```

Comment:
This error doesn't tell me anything

```suggestion
    if (ts.isNil(input)) throw new Error('Expected value not to be empty');
```